assets/src/data/persistence/activity.ts

```diff
@@ -246,7 +245,6 @@ export function edit(
       indexBibrefs(update.authoring, citationRefs);
     }
     update.content.bibrefs = citationRefs;
-    update = fixObjectiveParts(update) as ActivityUpdate & { releaseLock: boolean };
```

Comment:
This line was the cause of the bug, it should have been over in adaptive related saving routine instead.